unlock-app/src/hooks/useLockSettings.ts

```diff
@@ -70,3 +77,25 @@ export function useSaveSlugSetting() {
     }
   })
 }
+
+export function useSaveLockSettings() {
```

Comment:
Please make sure you replace all the other places where we have the same API call?

Reply:
i will move in a different PR